lib/big_ml/anomaly.rb

```diff
@@ -0,0 +1,31 @@
+require 'big_ml/base'
+
+module BigML
+  class Anomaly < Base
```

Comment:
Since this are new endpoint integration, probably make sense to add them into the integration specs to make sure they 're still working in some time form now.

https://github.com/joseribes/big_ml/tree/propertiesupdate/spec/integration